Backend/src/main/java/com/onbank/MockUser.java

```diff
@@ -0,0 +1,11 @@
+package com.onbank;
+
+import lombok.Data;
+import org.springframework.stereotype.Component;
+
+@Data
+@Component
+public class MockUser {
```

Comment:
A czemu to jest komponentem? Proponuje zrobić klasę User z polami username i accountNumber i w klasie Mocks zrobić statyczną metodę getMockUser() która zwróci nam stworzoną instancje z wypełnionymi polami

Reply:
Wiadomo że tutaj te pola się nie powinny zmieniać więc robienie beana singletona ma pewien sens ale na ogół nie tworzymy modeli jako beany.